ntbs-service/Pages/Shared/_NotificationLayout.cshtml

```diff
@@ -13,42 +13,93 @@
             <nhs-main-wrapper>
                 <nhs-grid-row>
                     <nhs-grid-column grid-column-width="Full">
-                        <h1 class="nhs-heading-1">NTBS #@Model.NotificationId - @Model.NotificationStatus </h1>
+                        <div class="@Model.Notification.NotificationBannerCssClass">
+                            <div class="notification-banner-title-container @Model.Notification.NotificationBannerTitleCssClass"> 
+                                <nhs-grid-row classes="notification-overview-details">
+                                    <nhs-grid-column grid-column-width="OneQuarter">
+                                        <div class="notification-banner-title-text"> @Model.Notification.NotificationStatusString </div>
+                                        <div class="notification-banner-title-text"> #@Model.Notification.NotificationId </div>
+                                    </nhs-grid-column>
+                                    <nhs-grid-column grid-column-width="OneQuarter">
+                                        <div class="notification-details-label"> Date of notification </div>

```

Comment:
And this stuff.. I'm wondering, if we want to use a definition list: https://css-tricks.com/utilizing-the-underused-but-semantically-awesome-definition-list/.. Not sure how happy it'll be with the different `div` interjecting the structures, but i think it's worth trying. Let's do some quick pairing on it?